sdks/python/apache_beam/options/pipeline_options_test.py

```diff
@@ -626,6 +626,14 @@ def test_lookup_experiments(self):
     self.assertEqual(
         True, debug_options.lookup_experiment('existing_experiment'))
 
+  def test_get_cloud_profiler_service_name(self):
```

Comment:
Let's test `None` and JOB_NAME case as well.  See for example https://stackoverflow.com/questions/31582750/python-mock-patch-os-environ-and-return-value/67477901#67477901 how to mock an environment variable.

Possible names for unit tests (ideally, test name includes tested scenario and expected result):

```
test_profiler_uses_provided_service_name_when_specified
test_profiler_uses_job_name_when_service_not_specified
```

Reply:
done